server/tasks/github_webhook_tasks.py

```
@@ -0,0 +1,988 @@
+        # ``redact_token`` covers any token-shaped substring that an
+        # exception message could echo back from a misbehaving handler
+        # (e.g. a downstream HTTP call that surfaces token in the body).
+        duration_ms = int((time.monotonic() - start) * 1000)
+        logger.exception(
+            "gh_webhook_handler=dispatch event_type=%s delivery_id=%s "
+            "status=failed duration_ms=%d error_class=%s msg=%s",
+            event_type,
+            delivery_id,
+            duration_ms,
+            type(exc).__name__,
+            redact_token(str(exc)),
```

Ruff (0.15.12)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)